chiselFrontend/src/main/scala/chisel3/CompileOptions.scala

```diff
@@ -52,7 +54,9 @@ object ExplicitCompileOptions {
     // Require an explicit DontCare assignment to generate a firrtl DefInvalid
     val explicitInvalidate: Boolean,
     // Should the reset type of Module be a Bool or a Reset
-    val inferModuleReset: Boolean
+    val inferModuleReset: Boolean,
+    // Should the top module reset, if inferred, be a Async instead of Bool?
+    val topAsyncReset: Boolean
```

Comment:
I would not mention the "if inferred". Whether inferred or not, the current behavior is that the top-reset is `Bool`. This is kind of an orthogonal parameter.

Reply:
but it is not my intenntion to override if someone actually has a module with a Bool reset (which might not even make sense now that i think about it... not sure how to specify that in Chisel)

Reply:
For this case, since it's only the top module, it is **only** the case that the top is a `Bool`, so I would argue that the current behavior is that this parameter already exists (implicitly), it's just always hardcoded to `false`.